packages/api/tests/endpoints/test-granules.js

```diff
@@ -2118,20 +2120,22 @@ test.serial('PATCH updates an existing granule in all data stores', async (t) =>
   );
 });
 
-test.serial('PATCH executes successfully with no non-required-field-updates (testing "inert" update/undefined fields)', async (t) => {
+test.serial('PATCH executes successfully with no non-required-field-updates (testing "insert" update/undefined fields)', async (t) => {
   const {
     esClient,
+    executionPgRecord,
```

Comment:
This is a fix from merging main into the parent branch, I'll make a similar commit to the parent branch shortly.